<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game Master's Notepad</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/nopic.png</normaloff>:/images/nopic.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="charactersTab">
       <attribute name="title">
        <string>Characters</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QListView" name="charactersListView">
            <property name="modelColumn">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditSearchCharacter"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="buttonAddNew">
              <property name="text">
               <string>Add new</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonRemove">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>485</width>
               <height>425</height>
              </rect>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4" stretch="2,3">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,6">
                <property name="sizeConstraint">
                 <enum>QLayout::SetDefaultConstraint</enum>
                </property>
                <item>
                 <widget class="ClickableImageLabel" name="labelImage">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="autoFillBackground">
                   <bool>false</bool>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::Box</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <property name="lineWidth">
                   <number>2</number>
                  </property>
                  <property name="midLineWidth">
                   <number>1</number>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="pixmap">
                   <pixmap resource="resources.qrc">:/images/nopic.png</pixmap>
                  </property>
                  <property name="scaledContents">
                   <bool>false</bool>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QFormLayout" name="formLayout">
                  <item row="0" column="0">
                   <widget class="QLabel" name="labelText">
                    <property name="text">
                     <string>Name</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="lineEditName"/>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="labelAge">
                    <property name="text">
                     <string>Age</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLineEdit" name="lineEditAge"/>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="labelRace">
                    <property name="text">
                     <string>Race</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QLineEdit" name="lineEditRace"/>
                  </item>
                  <item row="3" column="0">
                   <widget class="QLabel" name="labelProfession">
                    <property name="text">
                     <string>Profession</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1">
                   <widget class="QLineEdit" name="lineEditProfession"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,0,0,0">
                <item>
                 <widget class="QLabel" name="labelBackstory">
                  <property name="text">
                   <string>Backstory</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QTextEdit" name="textEditBackstory">
                  <property name="contextMenuPolicy">
                   <enum>Qt::DefaultContextMenu</enum>
                  </property>
                  <property name="textInteractionFlags">
                   <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextEditable|Qt::TextEditorInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelDescription">
                  <property name="text">
                   <string>Description</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QTextEdit" name="textEditDescription"/>
                </item>
                <item>
                 <widget class="QLabel" name="labelNotes">
                  <property name="text">
                   <string>Notes</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QTextEdit" name="textEditNotes"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="locationsTab">
       <attribute name="title">
        <string>Locations</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>807</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAboutApp"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAboutApp">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableImageLabel</class>
   <extends>QLabel</extends>
   <header>clickableimagelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>lineEditSearchCharacter</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>onCharactersSearchTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonRemove</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>removeCharacter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAddNew</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addCharacter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onCharactersSearchTextChanged(QString)</slot>
  <slot>removeCharacter()</slot>
  <slot>addCharacter()</slot>
 </slots>
</ui>
